packages/cli/src/commands/doctor/healthchecks/cocoaPods.ts

```diff
@@ -1,11 +1,93 @@
+import execa from 'execa';
+import chalk from 'chalk';
+import readline from 'readline';
+import wcwidth from 'wcwidth';
+import {logger} from '@react-native-community/cli-tools';
 import {checkSoftwareInstalled} from '../checkInstallation';
-import {installCocoaPods} from '../../../tools/installPods';
+import {
+  promptCocoaPodsInstallationQuestion,
+  runSudo,
+} from '../../../tools/installPods';
+import {brewInstall} from '../../../tools/brewInstall';
 import {HealthCheckInterface} from '../types';
 
+function calculateQuestionSize(promptQuestion: string) {
+  return Math.max(
+    1,
+    Math.ceil(wcwidth(promptQuestion) / (process.stdout.columns || 80)),
+  );
+}
+
+function clearQuestion(promptQuestion: string) {
+  readline.moveCursor(
+    process.stdout,
+    0,
+    -calculateQuestionSize(promptQuestion),
+  );
+  readline.clearScreenDown(process.stdout);
+}
+
 export default {
   label: 'CocoaPods',
   getDiagnostics: async () => ({
     needsToBeFixed: await checkSoftwareInstalled('pod'),
   }),
-  runAutomaticFix: async ({loader}) => await installCocoaPods(loader),
+  runAutomaticFix: async ({loader}) => {
+    loader.stop();
+
+    const {
+      installMethod,
+      promptQuestion,
+    } = await promptCocoaPodsInstallationQuestion();
+
+    // Capitalise `Homebrew` when printing on the screen
+    const installMethodCapitalized =
+      installMethod === 'homebrew'
+        ? installMethod.substr(0, 1).toUpperCase() +
+          installMethod.substr(1).toLowerCase()
```

Comment:
```suggestion
          installMethod.substr(1)
```
no need to lowercase already lowercased letters